src/components/Sections/ItemsSection.less

```diff
@@ -18,7 +18,7 @@
 
     &.card {
       .section-item-header {
-        padding-bottom: 9px;
+        padding-bottom: 6px;
```

Comment:
General comment,
About the multiple changes in padding, width, margin etc.
What is the root cause for these changes? (improvement, bug ...)

Reply:
Talked with Eynav and Hadas and they gave some comments